install.sh

```diff
@@ -90,17 +92,28 @@ download_and_install() {
   else
     version="${PNPM_VERSION}"
   fi
-  pkgName="@pnpm/${platform}-${arch}"
-  archive_url="https://registry.npmjs.org/${pkgName}/-/${platform}-${arch}-${version}.tgz"
+  if [ "${platform}" = "linuxstatic" ]; then
+    archive_url="https://github.com/pnpm/pnpm/releases/download/v${version}/pnpm-${platform}-${arch}"
+  else
+    pkgName="@pnpm/${platform}-${arch}"
+    archive_url="https://registry.npmjs.org/${pkgName}/-/${platform}-${arch}-${version}.tgz"
+  fi
+
```

Comment:
I guess we may download all tarballs from github. Not just the linux static one.

Reply:
this is indeed something possible, and this is what I have done in the past on the Symfony CLI installer.
But I didn't want to suggest such a significant change for a first contribution here 😅

Please note though, artifacts released are not tarballs but directly the binaries.